services/docker-compose.yml

```diff
@@ -113,6 +113,35 @@ services:
       - interactive_services_subnet
       - computational_services_subnet
 
+  static-webserver:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/static-webserver:${DOCKER_IMAGE_TAG:-latest}
+    environment:
+      - SERVER_HOST=0.0.0.0
+      - SERVER_PORT=80
+      - SERVER_LOG_LEVEL=error
+      - SERVER_ROOT=/static-content
+    env_file:
+      - ../.env
```

Comment:
SUGGESTION: Do you really need to load all the environments from .env? For the moment I think you use none of thse on .env, right?

Reply:
You are correct, none of these are used for this service.